gdk4/src/auto/texture.rs

```diff
@@ -76,11 +76,11 @@ impl<O: IsA<Texture>> TextureExt for O {
         unsafe { ffi::gdk_texture_get_width(self.as_ref().to_glib_none().0) }
     }
 
-    fn save_to_png(&self, filename: &str) -> bool {
+    fn save_to_png<P: AsRef<std::path::Path>>(&self, filename: P) -> bool {
```

Comment:
Needs to become 0.3 because of API changes @bilelmoussaoui (just so you don't forget :) )